src/Apps/Components/AppShell.tsx

```diff
@@ -25,6 +26,7 @@ export const AppShell: React.FC<AppShellProps> = props => {
 
   useRunAuthIntent()
   useAuthValidation()
+  useSocialAuthTracking()
```

Comment:
Since each of these hooks involve auth, any chance we could roll them up into some kind of `useAppAuth` hook and then just import that?

Reply:
@dzucconi - thoughts on this?

Reply:
Something like `useAfterAuth` or something would make sense, I guess, but I don't feel particularly strongly about it. I think the main gain of that organization would be the ability to mock them away in one.

Reply:
(I'll follow up.)